services/search/pkg/content/content.go

```diff
@@ -20,6 +21,7 @@ type Document struct {
 	Mtime    string
 	MimeType string
 	Tags     []string
+	Audio    *libregraph.Audio `json:"audio,omitempty"`
```

Comment:
Hmm. Does it make sense that `search` knows about `graph` service details? (I mean libregraph pkg) Also half of the `Audio` struct is not supported anywhere. Should we create a dedicated struct for the `Audio` fields we support?

Reply:
The alternative is to define a third struct with the exact same fields - we already have the libregraph model and we have the protobuf message. It would be cumbersome and not useful imho 🤷🏻 

We have this ADR that says, that we - for now - want to store exactly what is supported by the Graph API as a common denominator: https://github.com/owncloud/ocis/blob/master/docs/ocis/adr/0023-index-and-store-metadata.md

So I'd say it makes sense to use that model ...

Reply:
one downside is: I've been playing with geo/location data using the `libregraph.GeoCoordinates` model and that is not supported by `bleve.geo`. It would need to be transformed - but there's probably never an abstraction that works everywhere, we need to translate anyhow. If we adjust the location type (not part of this pr) to match bleve's expectations, it might not fit the next search engine that is implemented. not sure what to do.

Reply:
Ok makes sense. Then let's stick with this for now :+1: